code/modules/mob/living/carbon/emote.dm

```diff
@@ -265,7 +279,9 @@
 				if(sound_message)
 					message = sound_message
 				var/sound
-				switch(H.gender)
+				if (isvox(H) || isskelevox(H))
+					sound = pick(birb_sounds)
+				else switch(H.gender)
```

Comment:
One-line `else`

Reply:
Aright. Lemme fix that.

Reply:
Did this.

Reply:
```
else
    switch(H.gender)
```
Which will also require tabbing the contents of the `switch()`

Reply:
Similar as the above.